CodeConverter/VB/CommonConversions.cs

```diff
@@ -142,12 +142,15 @@ public static StatementSyntax CreateLocalDeclarationStatement(SyntaxTokenList sy
 
         private VariableDeclaratorSyntax ConvertToVariableDeclarator(CSS.DeclarationExpressionSyntax des)
         {
+            
             var id = ((IdentifierNameSyntax)des.Accept(_nodesVisitor)).Identifier;
             var ids = SyntaxFactory.SingletonSeparatedList(SyntaxFactory.ModifiedIdentifier(id));
             TypeSyntax typeSyntax;
             if (des.Type.IsVar) {
                 var typeSymbol = ModelExtensions.GetSymbolInfo(_semanticModel, des.Type).ExtractBestMatch<ITypeSymbol>();
-                typeSyntax = typeSymbol?.ToVbSyntax(_semanticModel, des.Type);
+                var csTypeSyntax =
+                   CS.SyntaxFactory.ParseTypeName(typeSymbol.ToMinimalDisplayString(_semanticModel, des.Type.SpanStart));
+                typeSyntax = (TypeSyntax)csTypeSyntax.Accept(_nodesVisitor);
```

Comment:
This class has a property called `VbSyntaxGenerator` which is a Roslyn class intended for converting a symbol to the appropriate syntax. Could you see if something like `VbSyntaxGenerator.TypeExpression(...)` achieves the same result

Reply:
I updated the code to use TypeExpression, replaced other call to ToVBSyntax and then removed ToVBSyntax.